src/components/menu/menu-full-screen/menu-full-screen.spec.tsx

```diff
@@ -173,51 +173,45 @@ const MockFullScreenMenuWithUpdatingItems = () => {
 };
 
 describe("MenuFullscreen", () => {
-  let wrapper: ReactWrapper;
-
-  beforeAll(() => {
-    loggerSpy.mockImplementation(() => {});
+  afterEach(() => {
+    // manually clean up DOM due to use of React portals
+    window.document.body.innerHTML = "";
   });
 
-  afterAll(() => {
-    loggerSpy.mockRestore();
+  it("by default, menu's root container should not exist", () => {
+    render(
+      <MenuFullscreen onClose={() => {}}>
+        <MenuItem href="#">Item one</MenuItem>
+      </MenuFullscreen>
+    );
+    expect(screen.queryByRole("dialog")).toBeNull();
   });
 
-  beforeEach(() => {
-    wrapper = render({});
+  it("has correct component, element and role data tags on root container when menu is open", () => {
+    render(<TestMenu isOpen />);
+    const menu = screen.getByRole("dialog");
+
+    expect(menu).toHaveAttribute("data-component", "menu-fullscreen");
```

Comment:
suggestion (non-blocking): you could use this [util](https://github.com/Sage/carbon/blob/947eaa1d8276dfcbf6ffc70e61d0f92e741d51e0/src/__internal__/utils/helpers/tags/tags-specs/tags-specs.ts#L25) if you want